src/AutoMapper/Tests/AutoMapperTest.php

```diff
@@ -632,6 +671,11 @@ public function testNoAutoRegister(): void
 
     public function testWithMixedArray(): void
     {
+        $configurationUser = $this->autoMapper->getMetadata(Fixtures\User::class, Fixtures\UserDTOProperties::class);
+        $configurationUser->forMember('properties', function (Fixtures\User $user) {
+            return $user->getProperties();
+        });
```

Comment:
I understand why you added it (same automapper for all test, custom metadata defined previously) but by doing this, you're not testing that the automapper is able to map a mixed array without ant specific config...
I would recommend either to move this tests before previous ones to ensure the automapper does not have custom config when ran or, even better, to create new Fixtures for your specific use case.

Reply:
Fixed, I made a new DTO and keep the old one for this test, thanks for your review !